sources/HUBViewModelDiff.m

```diff
@@ -37,6 +38,12 @@
     return [indexPaths copy];
 }
 
+@interface  HUBViewModelDiff ()
+
+@property (nonatomic, assign) BOOL headerComponentIdentifierHasChanged;
```

Comment:
I don't have strong opinions if to have two public APIs or one. Please decide if it is needed in scope of this PR as it can be of course changed later.